packages/backend/src/api/endpoints/SamlConnectionApi.ts

```diff
@@ -13,7 +13,9 @@ type SamlConnectionListParams = {
 type CreateSamlConnectionParams = {
   name: string;
   provider: SamlIdpSlug;
+  /** @deprecated Use `domains` array instead. This field will be removed in a future API version. */
```

Comment:
Nit: Multiple lines for JSDocs, but like a pattern than anything else.

```suggestion
  /** 
  * @deprecated Use `domains` array instead. This field will be removed in a future API version. 
  */
```